physics/Interstitials/UFS_SCM_NEPTUNE/GFS_rrtmgp_cloud_mp.F90

```diff
@@ -12,10 +12,11 @@ module GFS_rrtmgp_cloud_mp
   use rrtmgp_lw_cloud_optics, only: &
        radliq_lwr => radliq_lwrLW, radliq_upr => radliq_uprLW,&
        radice_lwr => radice_lwrLW, radice_upr => radice_uprLW  
-  use module_mp_thompson, only: calc_effectRad, Nt_c_l, Nt_c_o, re_qc_min, re_qc_max,  &
-       re_qi_min, re_qi_max, re_qs_min, re_qs_max
-  use module_mp_thompson_make_number_concentrations, only: make_IceNumber,             &
+  use module_mp_tempo_utils, only: calc_effectRad, make_IceNumber,         &
```

Comment:
@dustinswales Are you OK switching from Thompson's subroutines to TEMPO's permanently? There's no need to import both (Thompson and TEMPO) and use if statements in this scheme as done for RRTMG?